src/debuglogjs.h

```diff
@@ -0,0 +1,52 @@
+/*
+    Copyright (C) 2020 Sebastian J. Wolf and other contributors
+
+    This file is part of Fernschreiber.
+
+    Fernschreiber is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Fernschreiber is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Fernschreiber. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef DEBUGLOG_H
```

Comment:
Make is `DEBUGLOGJS_H` or something. I used `FERNSCHREIBER_DEBUG_LOG_H` in `debuglog.h` because `debuglog` is such a generic name that it's easy to imagine `#ifndef DEBUGLOG_H` appearing in one of the headers indirectly pulled in from elsewhere.